src/api/stats/matic/getGoldenbullLpApys.js

```diff
@@ -0,0 +1,33 @@
+const getMasterChefApys = require('./getMaticMasterChefApys');
+
+const MasterChefAbi = require('../../../abis/matic/GoldenbullMasterChef.json');
+const pools = require('../../../data/matic/goldenbullLpPools.json');
+const { quickClient } = require('../../../apollo/client');
+const { addressBook } = require('blockchain-addressbook');
+const {
+  polygon: {
+    platforms: { goldenbull },
+    tokens: { GBULL },
+  },
+} = addressBook;
+const { quickLiquidityProviderFee } = require('./getQuickLpApys');
+const {
+  getScientificNotationFromTokenDecimals,
+} = require('../../../utils/getScientificNotationFromTokenDecimals');
+
+const getGoldenbullLpApys = async () =>
+  await getMasterChefApys({
+    masterchef: goldenbull.masterchef,
```

Comment:
maybe we can keep masterChef address here? 
it seems not used anywhere else. 
ABI is here, LPs with all addresses are here, weird to update address-book package just for masterChef address, no?

Reply:
Well i was thinking about refactoring the data as well, and abi could be moved to address-book as well. There's checksum benefit as well in address-book. We could open up the discussion to others